pom.xml

```diff
@@ -6819,4 +6824,57 @@
 			</dependency>
 		</dependencies>
 	</dependencyManagement>
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.codehaus.mojo</groupId>
+				<artifactId>exec-maven-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>generate-gradle-catalog-and-platform</id>
+						<phase>generate-sources</phase>
+						<goals>
+							<goal>exec</goal>
+						</goals>
+						<configuration>
+							<executable>./gradlew</executable>
+							<workingDirectory>gradle-scijava</workingDirectory>
+							<arguments>
+								<argument>generateCatalogAndPlatform</argument>
+							</arguments>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
+			<!--		https://www.mojohaus.org/exec-maven-plugin/exec-mojo.html	-->
+			<plugin>
+				<groupId>org.codehaus.mojo</groupId>
+				<artifactId>build-helper-maven-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>attach-gradle-catalog</id>
+						<!--						Check if correct-->
+						<phase>package</phase>
```

Comment:
I think it works without adding the `<phase>` here at all. The default should be fine. See similar setup here:
https://github.com/google/guava/blob/master/guava/pom.xml#L205

Reply:
yep, I can confirm it works